cli/src/etos_client/client.py

```diff
@@ -106,20 +112,39 @@ def start(self, spinner):
         :rtype: bool
         """
         spinner.info(str(self.data))
-        generator = self.etos.http.retry(
-            "POST", f"{self.cluster}/etos", timeout=30, json=self.data
-        )
-        response = None
-        try:
-            for response in generator:
+
+        end_time = time.time() + 30
+        while time.time() < end_time:
+            try:
+                response = self.etos.http.request(
+                    "POST", f"{self.cluster}/etos", json=self.data
+                )
                 self.test_execution = response
                 self.test_suite_id = response.get("tercc")
                 self.artifact_id = response.get("artifact_id")
                 self.artifact_identity = response.get("artifact_identity")
                 self.event_repository = response.get("event_repository")
                 break
-        except ConnectionError as exception:
-            spinner.warn(str(exception))
+            except HTTPError as http_error:
+                response = http_error.response
+                if 400 <= response.status_code < 500:
+                    try:
+                        response_json = response.json()
+                    except JSONDecodeError:
+                        response_json = {"detail": "Unknown client error from ETOS"}
```

Comment:
Should we try to add the "raw" text body from the response as a hint to what might have gone wrong?